qa/erasure-code/ec-rados-plugin=jerasure-k=2-m=2-crush.yaml

```diff
@@ -1,13 +1,3 @@
-roles: 
-- [mon.a, mgr.y, osd.0]
-- [mon.b, osd.1] 
-- [mon.c, osd.2] 
-- [mgr.x, osd.3, client.0]
-overrides:
-  ceph:
-    conf:
-      osd:
-        osd shutdown pgref assert: true
```

Comment:
`../thrash/clusters` offers only `fixed-4.yaml` which is different from `k=2-m=2-crush` original `roles:`. Is it intentional?

Reply:
@Matan-B well. it creates those roles, it doesn't mean it will use them

Reply:
@Matan-B we are creating more roles than we need, but we are not using all of them in the case of k=2-m=2
erasure-code mention:
```
   erasure_code_profile:
      name: jerasure22profile
      plugin: jerasure
      k: 2
      m: 2
      technique: reed_sol_van
      crush-failure-domain: osd
      ```

Reply:
Ah, got it now.
Any other `qa/erasure-code/*.yaml` does not specify roles. So those should be removed from `ec-rados-plugin=jerasure-k=2-m=2-crush.yaml` and `ec-rados-plugin=jerasure-k=8-m=6-crush.yaml` respectively.

Reply:
> Any other `qa/erasure-code/*.yaml` does not specify roles. So those should be removed from `ec-rados-plugin=jerasure-k=2-m=2-crush.yaml` and `ec-rados-plugin=jerasure-k=8-m=6-crush.yaml` respectively.


This can be added to the commit message